src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/Dropdowns/IdTypes.json

```diff
@@ -0,0 +1,10 @@
+[{
```

Comment:
The types are described in AAS-Objects lib. Maybe the menus should reference this lib